drivers/storage/portworx/component/telemetry.go

```diff
@@ -99,6 +108,11 @@ func (t *telemetry) Delete(cluster *corev1.StorageCluster) error {
 		return err
 	}
 
+	err := k8sutil.DeleteConfigMap(t.k8sClient, TelemetryCCMProxyConfigMapName, cluster.Namespace, *ownerRef)
+	if err != nil && !errors.IsNotFound(err) {
```

Comment:
nit: `k8sutil.DeleteConfigMap` handles the `IsNotFound` error. We don't have to check it again here.